redfish-core/lib/ethernet.hpp

```diff
@@ -1103,20 +1103,91 @@ inline void
         dbus::utility::DbusVariantType(macAddress));
 }
 
+void deleteIPv4StaticAddresses(
+    const std::string& ethifaceId,
+    const std::shared_ptr<bmcweb::AsyncResp>& asyncResp)
+{
+    crow::connections::systemBus->async_method_call(
+        [asyncResp, ethifaceId](const boost::system::error_code errorCode,
+                                dbus::utility::ManagedObjectType& resp) {
+        if (errorCode)
+        {
+            messages::internalError(asyncResp->res);
```

Comment:
A LOG_ERROR here with the errorCode would be useful

Reply:
done